src/coreclr/pal/src/thread/process.cpp

```diff
@@ -2821,6 +2821,7 @@ CorUnix::InitializeProcessCommandLine(
     if (lpwstrFullPath)
     {
         LPWSTR lpwstr = PAL_wcsrchr(lpwstrFullPath, '/');
+        if (!lpwstr) {palError = ERROR_INTERNAL_ERROR; goto exit;}
```

Comment:
```suggestion
        if (!lpwstr)
        {
            ERROR("Invalid full path\n");
            palError = ERROR_INTERNAL_ERROR;
            goto exit;
        }
```

Reply:
Follow style of surrounding code

Reply:
Jan, thanks for the correction, we'll take it into account for the future.

Reply:
Corrected